### ISSUE TITLE
Updating from swagger-tools

### ISSUE THREAD

**Issue by alexandruluca** [apigee-127/sway#192]
In https://github.com/apigee-127/swagger-tools/issues/335 you are mentioning that swagger-tools are deprecate due to obvious reasons. Let's say I would like to update swagger-tools to using sway, how can I achieve this or what alternate maintained package can I used instead of swagger-tools?

Sway to me does not seem like a drop in replacement for swagger-tools but more like an internal api. Is there something similar to swagger-tools based on sway? If it's not clear what I mean, I am talking about swagger validator/metadata/securiry middleware provided by swagger-tools

Thanks

**Comment by whitlockjc** [comment]
You're right, `sway` is an API that provides everything _(and more)_ than `swagger-tools` but just not in all the same forms.  `swagger-tools` is an API, CLI and Connect middleware.  For the API, `sway` is already more than enough to replace `swagger-tools`.  For a CLI, there is a `sway`-based implementation called `oval`: https://github.com/whitlockjc/oval  As for the Connect Middleware, I started a project but it's not been updated in a while: https://github.com/whitlockjc/sway-connect  I would much rather see `sway-connect` continued than to backport `sway` into `swagger-tools`.  You interested in helping?
